core/services/chainlink/testdata/config-full.toml

```diff
@@ -143,16 +147,18 @@ DefaultTransactionQueueDepth = 17
 GasPriceBufferPercent = 12
 GasTipCapBufferPercent = 43
 BaseFeeBufferPercent = 89
-MaximumGracePeriod = 31
-RegistryCheckGasOverhead = 90
-RegistryPerformGasOverhead = 4294967295
-RegistryMaxPerformDataSize = 5000
-RegistrySyncInterval = '1h0m0s'
-RegistrySyncUpkeepQueueSize = 31
+MaxGracePeriod = 31
 TurnLookBack = 91
 TurnFlagEnabled = true
 UpkeepCheckGasPriceEnabled = true
 
+[Keeper.Registry]
+CheckGasOverhead = 90
+PerformGasOverhead = 4294967295
+MaxPerformDataSize = 5000
+SyncInterval = '1h0m0s'
+SyncUpkeepQueueSize = 31
```

Comment:
@jmank88 not sure what's the semantic meaning of `Keeper.Registry`, I would club them all under `Keeper`, they all are configurations of the node, not contract.

Aside: Many of these settings will be obsolete with keepers OCR and some new ones may be added. Not sure if a different struct `Keeper.OCR` would make sense

Reply:
Well I definitely don't want to imply that they are contract config fields, but everything here at the root level is already top level 'core' settings.
FWIW the old way could already be specified as:
```toml
Keeper.RegisterCheckGasOverhead = 90
```
Grouping under a table is just an alternative form:
```toml
[Keeper]
RegistryCheckGasOverhead = 90
```

So what I am proposing here is just:
```toml
Keeper.Register.CheckGasOverhead = 90
```
But could also be written as:
```toml
[Keeper]
Registry.CheckGasOverhead = 90
```
Or, as in this PR:
```toml
[Keeper.Registry]
CheckGasOverhead = 90
```


So I guess I am primarily thinking about groupings for readability and usability.

Reply:
I see, I think for now this looks good since all of these variables were sharing registry prefix anyway. 

We'll re-evaluate all of these in keeper v2.

Reply:
Thanks for taking a look :pray: 

FWIW another subtle difference is that the way the docs are configured, we can document the group/table itself when appropriate, so we can clarify anything we need to there. This PR will include that once I fill the rest out.